src/pages/inboxPlacement/tests/TestListPage.test.js

```diff
@@ -0,0 +1,46 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+import { tests } from './mockTests';
+
+import { TestListPage } from '../TestListPage';
+
+describe('Page: Test List', () => {
+
+  const subject = ({ ...props }) => {
+    const defaults = {
+      testsError: false,
+      testsPending: false,
+      tests: [],
+      listTests: jest.fn()
+    };
+    return shallow(<TestListPage {...defaults} {...props} />);
+  };
+
+  it('renders page correctly with defaults', () => {
+    const mockListTests = jest.fn();
+    const wrapper = subject({ listTests: mockListTests });
+
+    expect(wrapper).toMatchSnapshot();
+    expect(mockListTests).toHaveBeenCalled();
+  });
+
+  it('renders page with tests', () => {
+    const wrapper = subject({ tests });
+    expect(wrapper.find('Page')).toMatchSnapshot();
+  });
+
+  it('renders empty landing page when there are no tests', () => {
+    const wrapper = subject({ tests: []});
+    expect(wrapper.find('Page')).toMatchSnapshot();
```

Comment:
If you look at the new snapshot, it's hard to tell what exactly needs to be checked. Easier to just empty working like 

```suggestion
    expect(wrapper.find('Page').prop('empty')).toHaveProperty('show', true);
```

Reply:
fixed